sdk/src/types/block/output/nft.rs

```diff
@@ -618,3 +618,137 @@ pub mod dto {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
Am I too annoying if I ask you to move these to the `tests` folder for consistency? Or do they need private access or something?

Reply:
The dto one does need private access.

Reply:
Fair enough